src/sentry/hybridcloud/models/webhookpayload.py

```diff
@@ -79,7 +79,7 @@ def create_from_request(
         identifier: int | str,
         request: HttpRequest,
         integration_id: int | None = None,
-    ) -> Self:
+    ) -> WebhookPayload:
```

Comment:
due to an annoying variance issue interacting with a microbug in django-stubs it can't inference `cls.objects` properly here as also being `BaseManager[Self]`